shared/middleware/helpers/storage_apps.rb

```diff
@@ -57,14 +62,14 @@ def update(channel_id, value, ip_address)
     update_count = @table.where(id:id).exclude(state:'deleted').update(row)
     raise NotFound, "channel `#{channel_id}` not found" if update_count == 0
 
-    JSON.parse(row[:value]).merge(id: channel_id, isOwner: owner == @storage_id)
+    JSON.parse(row[:value]).merge(id: channel_id, isOwner: owner == @storage_id, createdAt: row[:created_at], updatedAt: row[:updated_at])
```

Comment:
Upon further investigation, I don't think we can remove the response data because the /projects page depends on the new value to be returned in the PUT response. See https://docs.angularjs.org/api/ngResource/service/$resource and grep for $save for details. If we decide to reduce the response, it may be possible to change the AngularJS code to work around this (grep for "custom method" on the $resource page above). For now it seems harmless to simply omit created_at from the response.